### ISSUE TITLE
Very minor typo in circular_shifts()

### ISSUE THREAD

**Issue by rhettinger** [more-itertools/more-itertools#897]
The loop variable is a double underscore but should be a single underscore.

```diff
-     for __ in repeat(None, n):
+     for _ in repeat(None, n):
```

**Comment by bbayles** [comment]
Just for my own knowledge, is there any reason other than aesthetics and/or consistency with the other throwaway variable names? I reflexively type `__` as a result of using the REPL and wanting to avoid colliding with the "last result," if only in my head.

**Comment by rhettinger** [comment]
There is no other reason. The norm is a single underscore for throw-away variables.

I haven't previously seen people type `__` in the REPL but that is a legit strategy to avoid stomping on the last result.

**Comment by bbayles** [comment]
Fixed in 4f4217c164eb99df5c09a3882b8c7222d7c74d6c
